packages/app/src/KubernetesDashboardPage.tsx

```diff
@@ -507,6 +507,140 @@ const NodesTable = ({
   );
 };
 
+const NamespacesTable = ({
+  where,
+  dateRange,
+}: {
+  where: string;
+  dateRange: [Date, Date];
+}) => {
+  const groupBy = ['k8s.namespace.name'];
+
+  const { data, isError, isLoading } = api.useMultiSeriesChart({
+    series: [
+      {
+        table: 'metrics',
+        field: 'k8s.pod.cpu.utilization - Gauge',
+        type: 'table',
+        aggFn: 'avg',
```

Comment:
I think it makes more sense to show the aggregate CPU and memory usage via a `sum` here